ui-manchette/src/components/Manchette.tsx

```diff
@@ -30,11 +30,12 @@ const Manchette = ({
   isProportional = true,
   toggleMode,
   children,
+  height,
 }: ManchetteProps) => (
   <div className="manchette-container">
     <div
       className="bg-white-100 border-r border-grey-30"
-      style={{ minHeight: `${INITIAL_OP_LIST_HEIGHT}px` }}
+      style={{ minHeight: `${INITIAL_OP_LIST_HEIGHT}px`, height }}
```

Comment:
as height can be undefined, i think you should go for 
`height : height | `${INITIAL_OP_LIST_HEIGHT}px``

Plus if we can change the height and make it lower than minHeight then we should remove minHeight